x-pack/plugins/observability/public/pages/rules/index.tsx

```diff
@@ -237,15 +235,14 @@ function RulesPage() {
         'data-test-subj': 'rulesTableCell-ContextStatus',
         render: (_enabled: boolean, item: RuleTableItem) => {
           return triggersActionsUi.getRuleStatusDropdown({
+            hideSnoozeOption: true,
```

Comment:
@Zacqary If you don't plan to add the notify column in o11y rules page as part of this PR, could you keep the old Snooze option for now? Then once your PR is merged, we can adapt our UI in the o11y Rules page.

Reply:
Please note that the Rule Details page uses the status component where the notify column could not be added.

Reply:
Sure, I can re-enable this for now. 

I wasn't planning on heavily adapting Observability implementations of the UI, just making sure existing functionality didn't break. In this PR I can go through all the broken stuff and make sure that it's not broken, but for new features or behaviors I may need y'all to take that on.

Reply:
@Zacqary Yep I agree. Just re-enable the snooze option in the o11y Rules page for now and then we will take care of adapting the snooze UI in observability.

What concerns me a bit is that my Kibana crashes even after I restarted my ES. While testing on your side, could you create an inventory rule type as well to see if you can reproduce the crashing?